index.html

```diff
@@ -105,31 +105,162 @@ <h2>Introduction</h2>
       </p>
 
       <section>
-        <h5>Relation to the Verifiable Credentials Data Model</h5>
-
+        <h5>Verifiable Credentials Data Model</h5>
         <p>
-          This specification defines encoding rules of the Verifiable Credential
-          Data Model onto JWT and JWS. It further defines processing rules for how
-          and when to make use of specific JWT-registered <a data-cite="rfc7519#section-4">claim</a> names and
-          specific JWS-registered header parameter names to allow systems based
-          on JWT to comply with this specification. If these specific
-          <a data-cite="rfc7519#section-4">claim</a> names and header parameters are present, their respective
-          counterpart in the standard <a data-cite="VC-DATA-MODEL#credentials">verifiable credential</a> and
-          <a data-cite="VC-DATA-MODEL#presentations">verifiable presentation</a> MAY be omitted to avoid duplication.
+          This specification describes how to secure media types 
+          expressing Verifiable Credentials and Verifiable Presentations 
+          as described in [[VC-DATA-MODEL]], using JWTs [[RFC7519]].
+        </p>
+        <p class="note">
+          The <code>application/vc+jwt</code> media type described in this specification defines
+          an example of a unidirectional mapping to a base media type defined in 
+          the [[VC-DATA-MODEL]]; see <a href="#example-mapping">Appendix A.4.1</a>.
         </p>
       </section>
+    </section>
 
-      
+
+
+    <section>
+      <h2>Securing JSON</h2>
+      <p class="note">
+        This representation relies on claims registered in the
+        <a href="https://www.iana.org/assignments/jwt/jwt.xhtml#claims">IANA JSON Web Token Claims Registry</a> whenever possible.
+      </p>
+      <p class="note">
+        Production of this representation does not use <code>vc+ld+json</code> as an input.
+      </p>
+      <section>
+        <h2>Credential Header</h2>
+        <p><a data-cite="rfc7519#section-5.1">typ</a> MUST use the media type <code>vc+jwt</code>.</p>
```

Comment:
`vc+jwt` isn't a media type. I don't know what the JWT spec calls this thing, but we can't call it a media type.

Reply:
`vc+jwt` is as much a media type as other requested media types... see also:

We are requesting this registration in the spirit of previous registrations:

- https://www.iana.org/assignments/media-types/application/secevent+jwt
- https://www.iana.org/assignments/media-types/application/at+jwt
- https://www.iana.org/assignments/media-types/application/logout+jwt
- https://www.iana.org/assignments/media-types/application/oauth-authz-req+jwt

<img width="629" alt="Screen Shot 2023-04-11 at 9 02 52 AM" src="https://user-images.githubusercontent.com/8295856/231188001-7c57e14e-ddc6-4601-afef-0f49d1a93307.png">


https://www.rfc-editor.org/rfc/rfc7519#section-5.1